lib/Minilla/Project.pm

```diff
@@ -485,8 +485,11 @@ sub extract_git_info {
     my $bugtracker;
     my $repository;
     my $homepage;
-    if ( `git remote show -n origin` =~ /URL: (.*)$/m && $1 ne 'origin' ) {
+    my $LANG=$ENV{LANG};
```

Comment:
El Dimecres 21 Octubre 2015, a les 08:33:35, Tokuhiro Matsuno va escriure:
> > @@ -485,8 +485,11 @@ sub extract_git_info {
> >      my $bugtracker;
> >      my $repository;
> >      my $homepage;
> > -    if ( `git remote show -n origin` =~ /URL: (.*)$/m && $1 ne 'origin' ) {
> > +    my $LANG=$ENV{LANG};
> 
> Could you use  'local'?
> 
Yes, thanks for the idea, I rarely need to use local on my code so I didn't think to solve the problem that way, the code looks clear now and the possible side effects more confined. 

I committed the code on my branch and it shall be on the PR now.

Thanks for your time,

jluis